x/axelarnet/keeper/keeper.go

```diff
@@ -113,6 +114,34 @@ func (k Keeper) GetCosmosChainByName(ctx sdk.Context, chain nexus.ChainName) (ty
 	return value, true
 }
 
+// setChainByIBCPath sets the chain name for the given ibc path
+func (k Keeper) setChainByIBCPath(ctx sdk.Context, ibcPath string, chain nexus.ChainName) error {
+	// register a cosmos chain to axelarnet
+	return k.getStore(ctx).SetNewValidated(ibcPathPrefix.Append(key.FromStr(ibcPath)),
+		utils.WithValidation(&gogoprototypes.StringValue{Value: chain.String()},
```

Comment:
I'll be switching this to a separate proto that stores both the chain name and ibc path